src/motors.cpp

```diff
@@ -1,28 +1,40 @@
 
 #include "motors.hpp"
+#include "ESC.h"
 
 namespace bzzz
 {
     static bool s_armStatus = false;
+    
+    ESC frontLeftEsc(FRONT_LEFT_ESC_PIN, ABSOLUTE_MIN_PWM, ABSOLUTE_MAX_PWM, ARM_ROTOR_SPEED); // ESC_Name (PIN, Minimum Value, Maximum Value, Arm Value)
+    ESC frontRightEsc(FRONT_RIGHT_ESC_PIN, ABSOLUTE_MIN_PWM, ABSOLUTE_MAX_PWM, ARM_ROTOR_SPEED);
+    ESC backLeftEsc(BACK_LEFT_ESC_PIN, ABSOLUTE_MIN_PWM, ABSOLUTE_MAX_PWM, ARM_ROTOR_SPEED);
+    ESC backRightEsc(BACK_RIGHT_ESC_PIN, ABSOLUTE_MIN_PWM, ABSOLUTE_MAX_PWM, ARM_ROTOR_SPEED);
 
-    void attachMotors(void){
-        // todo
-    };
-
-    void writeSpeedToEsc(void){
-        // todo
+    void writeSpeedToEsc(int rotor_speed_front_left, int rotor_speed_front_right, int rotor_speed_back_left, int rotor_speed_back_right)
```

Comment:
Do we need some sort of clipping here to prevent very high speeds?

Reply:
and to prevent negative speeds too

Reply:
That is taken care of by the RC_ESC library. In the `motors.cpp` file when the object is created for each ESC, the correct pin, the maximum and minimum PWM and the arm speed are assigned. The `ESC.cpp` from the RC_ESC library then does the clipping based of the maximum and minimum PWM